ClientApp/src/components/admin/common/GiverTable.tsx

```diff
@@ -14,7 +14,7 @@ import {
   FiberManualRecord,
 } from "@material-ui/icons";
 import React from "react";
-import { GiverType } from "../../../common/components/Types";
+import { GiverType } from "../../../components/shared/Types";
```

Comment:
Adding "baseUrl: 'src'" to the tsconfig.json file in ./ClientApp like this:
```json
{
  "compilerOptions": {
    "baseUrl": "src",
```
Will make "src" as an absolute path for any .tsx file in the ClientApp directory and will eliminate all the backtracking to shared component from: 
```tsx
import { GiverType } from "../../../components/shared/Types";
```
to
```tsx
import { GiverType } from "components/shared/Types";
```
Thus making the imports more readable and easier to follow

Reply:
This did require a restart to VSCode for the code to compile

Reply:
Think this will be a good feature, cleaning up the other 40+ files in this PR. 🥇

Reply:
Perfect suggestion for refactoring🥇

Reply:
Implemented in current pr